index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Mobile Version Skeletun</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav>
+    <ul class="bar">
+      <li>John Doe</li>
+      <li><img src="Union.png" alt="image of three rows"></li>
+    </ul>
+  </nav>
```

Comment:
- [ ] The container of this section should be a header tag. That way it is more descriptive and makes good use of semantic tags